pkg/kubectl/cmd/logs_test.go

```diff
@@ -196,7 +196,7 @@ func TestLogComplete(t *testing.T) {
 		{
 			name:     "No args case",
 			flags:    map[string]string{"selector": ""},
-			expected: "'logs (POD | TYPE/NAME) [CONTAINER_NAME]'.\nPOD or TYPE/NAME is a required argument for the logs command",
+			expected: "'logs [-f] [-p] (POD | TYPE/NAME) [-c CONTAINER]'.\nPOD or TYPE/NAME is a required argument for the logs command",
```

Comment:
Honestly, these tests are worthless, they don't give any value. We don't guarantee output stability and this is testing just that. Please remove both.